src/Workspaces/CSharp/Portable/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -572,6 +565,44 @@ private static bool AtEndOfIncompleteStringOrCharLiteral(SyntaxToken token, int
             {
                 selectedMembers.Free();
             }
+
+            bool IsSelectedFieldOrProperty(MemberDeclarationSyntax member)
```

Comment:
please make static.  also, ahve a return statement above it.  add the line `// local funcitons' below that return statement.

Reply:
note: all of the above makes it clear that his is just a helper method pulled into the current method.  it also makes it clear there's no procedural logic outside of htis that someone needs to scan for.

Reply:
@CyrusNajmabadi, I changed it to static and added the comment above it.  Since my return is in a try block, I don't think there's any way to put the return right above the local method.

Reply:
can you rewrite hte above as:

```c#
                var members = containingType.Members;
                var fieldIndex = members.IndexOf(firstMember);
                if (fieldIndex < 0)
                {
                    return ImmutableArray<MemberDeclarationSyntax>.Empty;
                }


            var selectedMembers = ArrayBuilder<MemberDeclarationSyntax>.GetInstance();

                for (var i = fieldIndex; i < members.Count; i++)
                {
                    var member = members[i];
                    if (IsSelectedFieldOrProperty(member))
                    {
                        selectedMembers.Add(member);
                    }
              }

                return selectedMembers.ToImmutable();

               // local functions

               //...
```

```

Reply:
@CyrusNajmabadi , thanks for the help!  I've made the changes and once tests pass I'm planning to merge.